example/rdma-test-pod2.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: rdma-test-pod-2
+  annotations:
+    k8s.v1.cni.cncf.io/networks: rdma-net-ipam
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: ubuntu00
+  restartPolicy: OnFailure
+  containers:
+  - image: mellanox/rping-test 
+    name: rdma-test-ctr
+    securityContext:
+      capabilities:
+        add: [ "IPC_LOCK","NET_ADMIN" ]
```

Comment:
why NET_ADMIN?

Reply:
if you are not using a cluster IPAM on secondary network and are assigning manually.

Today we dont have an official solution for secondary network IPAM what is cluster unique right ?

there are two net-attach-def CRs in the example, one with whareabouts IPAM and one without.

Reply:
why not static cni? https://github.com/containernetworking/plugins/tree/master/plugins/ipam/static
You can assign with it in the pod spec. Why we need to elevate the pod with  NET_ADMIN just to put static ip address?

Reply:
i was not aware of this CNI.
This is a better alternative than configuring within the pod which requires additional caps.

will modify